bucket/graalvm-lts.json

```diff
@@ -0,0 +1,25 @@
+{
+    "description": "High-performance, embeddable, polyglot Virtual Machine for JVM-langs (Java, Scala, Kotlin), JavaScript/NodeJS, Python, Ruby, R, and LLVM-langs (C, C++, Rust)",
+    "version": "20.3.1.2",
+    "homepage": "https://graalvm.org",
+    "license": "GPL-2.0",
+    "url": "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-20.3.1.2/graalvm-ce-java11-windows-amd64-20.3.1.2.zip",
+    "hash": "a4d115d81e0680bd2aabfe33d6c53f6f7b646ce9fa21568d533e9afb70a0de31",
+    "extract_dir": "graalvm-ce-java11-20.3.1.2",
+    "env_add_path": "bin",
+    "env_set": {
+        "JAVA_HOME": "$dir",
+        "GRAALVM_HOME": "$dir"
+    },
+    "checkver": {
+        "url": "https://github.com/graalvm/graalvm-ce-builds/releases",
+        "regex": "graalvm-ce-builds/releases/tag/vm-(20[\\d\\w.-]+)"
+    },
+    "autoupdate": {
+        "url": "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-$version/graalvm-ce-java11-windows-amd64-$version.zip",
```

Comment:
Maybe update this so Java version is also captured, next Java LTS is around the corner, I presume they will make binaries for that too. If the user want to force Java 11, there is the -jdk11 manifest for that?

Reply:
I am not sure how we will proceed with that

Do you have any recommendations?

Reply:
Ok, I'll add an issue, to make adjustments later.

Reply:
One way I found was

```
    "checkver": {
        "url": "https://github.com/graalvm/graalvm-ce-builds/releases",
        "regex": "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-(?<version>21[\\d\\w.-]+)/graalvm-ce-java(?<path>[\\d]+)-windows-amd64-[\\d\\w.-]+.zip"
    },
```

But this is dependent on order and will break during the 20s/10s transition

Reply:
`(?<jdk>1[\\d]+)` maybe?

Reply:
Yeah but that would lead to issues like 11 always selected instead of 15 due to the ascending order

I guess manual is safer

When 15 comes out, we anyways would have to add graalvm-lts-jdk15

Reply:
You can always use `"reverse": true`, ref: [Special Cases](https://github.com/lukesampson/scoop/wiki/App-Manifest-Autoupdate#special-cases)